global.json

```diff
@@ -6,7 +6,8 @@
       "components": [
         "Microsoft.VisualStudio.Component.VSSDK"
       ]
-    }
+    },
+    "xcopy-msbuild": "16.0.0-rc1-alpha"
```

Comment:
What's going on here?

Reply:
Build caught fact RoslynTools.MSBuild package version 16.0.0-alpha is no longer available. The other queue must contain agents that already have that version installed.